actionpack/lib/action_controller/metal/instrumentation.rb

```diff
@@ -63,7 +63,7 @@ def process_action(*)
           headers: request.headers,
           format: request.format.ref,
           method: request.request_method,
-          path: request.fullpath
+          path: request.filtered_path
```

Comment:
For now it's a direct replace, but we can consider adding a new key as well, to allow applications to choose/use whatever they need (& allowing for backward compatibility).